compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/gestures/UikitScrollable.uikit.kt

```diff
@@ -19,8 +19,15 @@
 package androidx.compose.foundation.gestures
 
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.input.pointer.PointerEvent
+import androidx.compose.ui.unit.Density
+import androidx.compose.ui.unit.IntSize
 
 @Composable
-internal actual fun platformScrollConfig(): ScrollConfig =
-    TODO("implement native platformScrollConfig")
+internal actual fun platformScrollConfig(): ScrollConfig = UiKitConfig
+
+private object UiKitConfig : ScrollConfig {
+    override fun Density.calculateMouseWheelScroll(event: PointerEvent, bounds: IntSize): Offset =
```

Comment:
Or iOS doesn't support mouse?

Reply:
I tried it locally. Basically, I seems that you can't scroll using the mouse wheel (or two fingers) on iOS simulator. E.g. no events received here at all. I have also checked other apps, they all do not scroll via the wheel (touchpad).

Reply:
Probably worth to copy-paste the implementation from androidMain. In case if this will be used in tests, which run on iOS. We can pass scroll events to them.

Reply:
Done